python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -152,6 +152,24 @@ def __init__(
         # Add required callbacks for ATCamera
         self.camera.evt_shutterDetailedState.callback = self.shutter_monitor_callback
 
+        # Create a remote for the ESS temperature
+        self.ess_remote = Remote(
+            self.domain,
+            "ESS",
+            index=201,
+            include=[
+                "temperature",
+            ],
+        )
+
+        # Add required callbacks for ESS
+        self.ess_remote.tel_temperature.callback = self.handle_temperature_callback
+        self._temperature_data = None
+
+        # Default temperature and max telemetry age
+        self.fallback_temperature = 10.0
```

Comment:
Are you sure this is the correct fallback temperature? Looking at the plots you sent me I would expect something around 15C. This should be a value that would fallback to what we have now without the temperature correction.